confgenerator/feature.go

```diff
@@ -0,0 +1,176 @@
+package confgenerator
+
+import (
+	"errors"
+	"fmt"
+	"reflect"
+	"strings"
+)
+
+var ErrTrackingInlineStruct = errors.New("cannot have tracking on inline struct")
+
+type Feature struct {
```

Comment:
Please document the exported items in this package, because I can't tell how any of this is expected to be used just from the field and function names.

Reply:
Changed in: [0f93735](https://github.com/GoogleCloudPlatform/ops-agent/pull/853/commits/0f937356b0edf460a8aa421e4e0398dfc4c018f8)